src/app/ResubscribePolicyDelegate.h

```diff
@@ -0,0 +1,80 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <lib/core/CHIPCore.h>
+#include <lib/support/FibonacciUtils.h>
+
+namespace chip {
+namespace app {
+/**
+ * @brief An ResubscribePolicyDelegate is used to configure resubscribe policy. Run expect to input the number of retires as seed,
```

Comment:
"Used to specify the re-subscription policy. Namely, the main method `Run` is invoked and provided the number of retries that have occurred so far, and whether and how long into the future a re-subscription should happen."

Reply:
Updated, thanks